app/main.py

```diff
@@ -1,5 +1,61 @@
+from __future__ import annotations
+
+
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
+        self.weight = weight
+
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
         self.weight = weight
+        self.coords = coords or [0, 0]
+
+    def go_forward(self, step: int = 1) -> None:
+        self.coords[1] += step
+
+    def go_back(self, step: int = 1) -> None:
+        self.coords[1] -= step
+
+    def go_right(self, step: int = 1) -> None:
+        self.coords[0] += step
+
+    def go_left(self, step: int = 1) -> None:
+        self.coords[0] -= step
+
+    def get_info(self) -> str:
+        return f"Robot: {self.name}, Weight: {self.weight}"
+
+
+class FlyingRobot(BaseRobot):
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        super().__init__(name, weight)
+        self.coords = coords or [0, 0, 0]
+
+    def go_up(self, step: int = 1) -> None:
+        self.coords[2] += step
+
+    def go_down(self, step: int = 1) -> None:
+        self.coords[2] -= step
+
+
+class DeliveryDrone(FlyingRobot):
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 max_load_weight: int,
+                 current_load: Cargo or None,
+                 coords: list = None
+                 ) -> None:
```

Comment:
fix identation also here 


```suggestion
    def __init__(
                 self,
                 name: str,
                 weight: int,
                 max_load_weight: int,
                 current_load: Cargo or None,
                 coords: list = None
                 ) -> None:
```

Reply:
Alright, thank you!